olm/update-nightly-bundle.sh

```diff
@@ -127,7 +127,16 @@ do
     incrementNightlyVersion "${platform}"
   fi
 
-  cp -rf "${ROOT_PROJECT_DIR}/deploy/crds/org_v1_che_crd.yaml" "${bundleFolder}/manifests"
+  templateCRD="${ROOT_PROJECT_DIR}/deploy/crds/org_v1_che_crd.yaml"
+  platformCRD="${bundleFolder}/manifests/org_v1_che_crd.yaml"
+
+  cp -rf $templateCRD $platformCRD
+  if [[ $platform == "openshift" ]]; then
```

Comment:
Not even sure it is *only* for OpenShift. I assume that it would also work on not-too-old versions of Kubernetes with OLM installed.

Reply:
Maybe, but I'm afraid to break something.

Reply:
> Not even sure it is _only_ for OpenShift. I assume that it would also work on not-too-old versions of Kubernetes with OLM installed.

By the way, adding this also for the Kubernetes case can also be done in a future PR (if we create a new issue to track it). So let me approve this PR to go faster.